tracer/test/Datadog.Trace.IntegrationTests/ContainerTaggingTests.cs

```diff
@@ -52,15 +41,14 @@ public async Task Http_Headers_Contain_ContainerId()
 
                 await tracer.FlushAsync();
 
-                var spans = agent.WaitForSpans(1);
-                Assert.Equal(1, spans.Count);
-                Assert.Equal(expectedContainedId, actualContainerId);
+                var spans = agent.WaitForSpans(count: 1);
+                Assert.Equal(expected: 1, spans.Count);
 
-                if (EnvironmentTools.IsWindows())
-                {
-                    // we don't extract the containerId on Windows (yet?)
-                    Assert.Null(actualContainerId);
-                }
+                // we don't extract the containerId on Windows (yet?)
+                var expectedContainedId = EnvironmentTools.IsWindows() ? null : ContainerMetadata.GetContainerId();
```

Comment:
Checking `EnvironmentTools.IsWindows()` is not needed here since `ContainerMetadata.GetContainerId()` will return `null` on Windows.

`EnvironmentTools.IsWindows()` was used before to decide if we should assert that `actualContainerId` was null (i.e. if the http header was not present).

Reply:
Fixed in 95739c93db893556b0ae72a955b2978424fa9ccd